app/gui/src/main/scala/org/bitcoins/gui/GlobalData.scala

```diff
@@ -67,12 +66,12 @@ object GlobalData {
   }
 
   /** Builds a url for the blockstream explorer to view the tx */
-  def buildTxUrl(txid: DoubleSha256DigestBE): String = {
+  def buildTxUrl(txIdHex: String): String = {
     network match {
       case MainNet =>
-        s"https://blockstream.info/tx/${txid.hex}"
+        s"https://blockstream.info/tx/${txIdHex}"
       case TestNet3 =>
-        s"https://blockstream.info/testnet/tx/${txid.hex}"
+        s"https://blockstream.info/testnet/tx/${txIdHex}"
```

Comment:
might be worth switching to mempool.space, they have a signet explorer and have moved to become the default explorer over blockstream.info imo